### DISCUSSION TITLE
如何设置默认模型

### DISCUSSION THREAD

**Discussion by connermo** [ChatGPTNextWeb/NextChat#4603] [Q&A]
如题，如何设置默认使用的模型，现在默认是gpt3.5-turbo，如何改成gpt-3.5-turbo-1106? 谢谢。

**Comment by code-october** [comment]
现在已经支持环境变量设置默认模型了

**Comment by connermo** [comment]
请问是哪个变量？应该如何设置，谢谢！

**Reply by QAbot-zh** [reply]
https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/issues/3891#issuecomment-2118761537

**Comment by connermo** [comment]
收到，谢谢!
